contrib/localnet/nginx/nginx.conf

```diff
@@ -0,0 +1,102 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+
+    upstream lcd_servers {
+        server cnclinode0:1317; #IP:Port of my application
+    }
+
+    server {
+        listen          7123;
+        server_name     lcd.commercio.network;
+        rewrite ^/stake/validators$ /staking/validators last;
+    
+        location / {
+            proxy_pass http://lcd_servers;
+            
+            proxy_set_header Host $host;
+            proxy_redirect off;
+
+            if ($request_method = 'OPTIONS') {
+                add_header 'Access-Control-Allow-Origin' '*';
+                add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
+                #
+                # Custom headers and headers various browsers *should* be OK with but aren't
+                #
+                add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
+                #
+                # Tell client that this pre-flight info is valid for 20 days
+                #
+                add_header 'Access-Control-Max-Age' 1728000;
+                add_header 'Content-Type' 'text/plain; charset=utf-8';
+                add_header 'Content-Length' 0;
+                return 204;
+            }
+            if ($request_method = 'POST') {
+                add_header 'Access-Control-Allow-Origin' '*';
+                add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
+                add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
+                add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
+            }
+            if ($request_method = 'GET') {
+                add_header 'Access-Control-Allow-Origin' '*';
+                add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
+                add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
+                add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
+            }
+        }
+    }
+    upstream websocket {
+        server cndnode0:26657;	
+    }
+
+    server {
+        listen 			7124;
+        server_name 	rpc.commercio.network;
```

Comment:
Is this server name correct?

Reply:
I will change the name to "rpc.commerce.local". In any case the configuration can be changed after the stack deploy